index.js

```diff
@@ -68,7 +68,9 @@ function setup(fetch) {
 		fetch = require('node-fetch');
 	}
 
-	fetch = fetch.default || fetch;
+	fetch = fetch.default
+		? Object.assign(fetch.default, fetch) // combines Headers with "fetch.default" function
```

Comment:
```suggestion
		? Object.assign({}, fetch.default, fetch) // combines Headers with "fetch.default" function
```

I don't think you want to override `fetch.default` so I suggest assigning to a new object.

Reply:
In this case `fetch` becomes an object and is not callable anymore. I've updated the PR if you don't want to decorate the original function.